packages/voting/manifest/templates/subgraph.template.yaml

```diff
@@ -10,26 +10,6 @@ dataSources:
 {{#VotingAncillaryDataSources}}
   {{> VotingAncillary.template.yaml}}
 {{/VotingAncillaryDataSources}}
-  - kind: ethereum/contract
-    name: VotingToken
-    network: {{network}}
-    source:
-      address: "{{votingTokenAddress}}"
-      abi: VotingToken
-      startBlock: {{votingTokenStartBlock}}
-    mapping:
-      kind: ethereum/events
-      apiVersion: 0.0.4
-      language: wasm/assemblyscript
-      file: ./src/index.ts
-      entities:
-        - User
-      abis:
-        - name: VotingToken
-          file: ./abis/VotingToken.json
-      eventHandlers:
-        - event: Transfer(indexed address,indexed address,uint256)
-          handler: handleTransferVotingToken
```

Comment:
No need to index this. Previously we were indexing this to keep track of UMA balances but we can query this in many ways easily